src/sentry/workflow_engine/endpoints/serializers.py

```diff
@@ -352,13 +352,18 @@ def get_attrs(self, item_list, user, **kwargs) -> MutableMapping[Workflow, dict[
         for wdcg in wdcg_list:
             dcg_map[wdcg.workflow_id].append(serialized_condition_groups[wdcg.condition_group_id])
 
+        detectors_map = defaultdict[int, list[str]](list)
+        for dw in DetectorWorkflow.objects.filter(workflow__in=item_list):
+            detectors_map[dw.workflow_id].append(str(dw.detector_id))
```

Comment:
A simpler way to write this is:
```py
DetectorWorkflow.objects.filter(workflow__in=item_list).values_list("detector_id", flat=True)
```
That will also limit the select in the db query, so it's only fetching the list of detector ids making it a little quicker.

Reply:
fixed in https://github.com/getsentry/sentry/pull/91998/commits/293175131f787e2a5fe8455ec63de26706260957 !